src/mumble/Log.cpp

```diff
@@ -374,7 +370,8 @@ QString Log::imageToImg(QImage img) {
 		imgwrite.write(img);
 	}
 
-	while ((qba.length() >= 65536) && (quality > 0)) {
+	const int safety = 1024;
```

Comment:
This code warrants some comments.
What is safety?
What is the "length * 2 + safety" expression?